content: Raph Overview & Specification update (#92)
fictionlab/docs: docs/raph-rover/documentation/specification.mdx

| Ground clearance        |                       80 mm                       |
| Wheel diameter          |                      170 mm                       |

  </FlexTableItem>
  <FlexTableItem style={{width:'55%'}}>

<ThemedImageZoom
  alt="Main dimensions of the Raph Rover"
  sources={{
    light: useBaseUrl(
      '/img/robots/raph/specification/raph_dimensions_black.webp',
    ),
    dark: useBaseUrl(
      '/img/robots/raph/specification/raph_dimensions_white.webp',
    ),
  }}
  width="3000"
  height="4997"
  figStyle={{
    width: 600,
  }}
/>

  </FlexTableItem>
</FlexTable>

### Speed and performance

<CenterContent>

| Parameter                     |    Value    |
| ----------------------------- | :---------: |
| Maximum linear speed          |   1.8 m/s   |
| Maximum angular speed         |  4.7 Rad/s  |
| Climb grade (no payload)      | 30° ( ~58%) |
| Climb grade with 5kg payload  | 20° ( ~36%) |
| Climb grade with 10kg payload | 10° ( ~18%) |
| Hill grade traversal          | 30° ( ~58%) |
| Nominal torque                |    4 Nm     |
| Maximum torque                |    8 Nm     |

</CenterContent>

### Drivetrain

> Raph rover is equipped with 2, independent steering wheels on the rear of the

Reviewer: ```suggestion
Raph rover is equipped with two independent steering wheels on the rear of the
```

| LED strip           |    4     | 143 RGBW Individually addressable LEDs. Used as rover status indicator as well as any custom purpose programmed by the user.<br />- Form factor: 3535<br />- Density: 144 LEDs/m<br />- Power: 23 W<br />- Temperature: 6000-6500K.<br />- Luminance R: 46; G: 156; B: 34; W:409 lm                                                                                                                                  |

</CenterContent>

#### Component's diagram

<ThemedImageZoom
  alt="Hardware diagram of Raph Rover"
  sources={{
    light: useBaseUrl(
      '/img/robots/raph/specification/raph-hardware-diagram-light.webp',
    ),
    dark: useBaseUrl(
      '/img/robots/raph/specification/raph-hardware-diagram-dark.webp',
    ),
  }}
  width="2000"
  height="2037"
  caption="Block diagram of Raph Rovers components"
  figStyle={{ paddingBottom: '2rem' }}
/>

#### Battery

Raph rover's batteries are enclosed in specially designed case, allowing them to
be quick swappable. Each pack has its own BMS that manages charging, discharging

> and has built-in protections

Reviewer: ```suggestion
and has built-in protections.
```

FreeRTOS and MicroROS to control rover tasks such as:

- Drivetrain control
- Power system management
- LED control
- IMU operation
- Odometry tracking

<ThemedImageZoom
  alt="External connections of RaphCore"
  sources={{
    light: useBaseUrl(
      '/img/robots/raph/specification/RaphCore-overview-black.webp',
    ),
    dark: useBaseUrl(
      '/img/robots/raph/specification/RaphCore-overview-white.webp',
    ),
  }}
  width="2100"
  height="964"
  caption="RaphCore PCB connectors"
/>

<CenterContent>

| Component                 |                                                                        Model                                                                         | Description                                                                                                                                       |
| ------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------ |

> | Microcontroller           |                                                                    STM32H5 series                                                                    | CPU type: 32-bit Arm® Cortex®-M33<br />CPU frequency: 250 MHz<br />Flash memory: 2 Mb                                                           |

Reviewer: ```suggestion
| Microcontroller           |                                                                    STM32H5 series                                                                    | CPU type: 32-bit Arm® Cortex®-M33<br />CPU frequency: 250 MHz<br />Flash memory: 2 MB                                                           |
```

| IMU                       |                                                                          -                                                                           | Sensor Type: 6-axis<br />Acceleration: 2 g, 4 g, 8 g, 16 g<br />Resolution: 16 bit<br />Sensitivity: 2097.2 deg/s/LSB                             |
| Internal power connectors | 3 x [MC&nbsp;1,5/&nbsp;2&#8209;G&#8209;3,5](https://www.phoenixcontact.com/pl-pl/produkty/obudowa-podstawowa-plytki-drukowanej-mc-15-2-g-35-1844210) | Additional power outputs for the rover. Each individual output can be disabled or enabled separately.                                             |
| External power connectors | 1 x [MC&nbsp;1,5/&nbsp;6&#8209;G&#8209;3,5](https://www.phoenixcontact.com/pl-pl/produkty/obudowa-podstawowa-plytki-drukowanej-mc-15-6-g-35-1844252) | Power outputs for the rover interface panel. Each individual output can be disabled or enabled separately. [See&nbsp;more](#external-power-panel) |

</CenterContent>

## Payload

The Raph Rover is engineered for easy and versatile payload integration. If your
sensor or hardware is ROS 2-compatible and falls within the rover's power (5V

> 4A, 12V 4A, and 24V 4A) and weight limits (10kg), you can be confident in its

Reviewer: ```suggestion
4A, 12V 4A, and 24V 4A) and weight limits (10 kg), you can be confident in its
```

| ---------- | :--------------------: | :--------------------------: |
| USB type A | USB 3.2 Gen 1 (5 Gb/s) | UPBoard 7000 (Main computer) |
| USB type C | USB 3.2 Gen 1 (5 Gb/s) | UPBoard 7000 (Main computer) |
| HDMI       |       HDMI 1.4b        | UPBoard 7000 (Main computer) |
| RJ-45      |    10/100/1000 Mb/s    |  Teltonika RUTX10 (Router)   |

</CenterContent>

### External power panel

<CenterContent>

|    Voltage    | Max amperage | Max power | Connector                                                                                                                             |
| :-----------: | :----------: | :-------: | :------------------------------------------------------------------------------------------------------------------------------------ |
|      5 V      |     4 A      |   20 W    | [Phoenix Contact MC 1,5/ 2-ST-3,5](https://www.phoenixcontact.com/pl-pl/produkty/wtyk-plytki-drukowanej-mc-15-2-st-35-1840366)        |
|     12 V      |     4 A      |   48 W    | [Phoenix Contact MC 1,5/ 2-ST-5,08](https://www.phoenixcontact.com/pl-pl/produkty/wtyk-plytki-drukowanej-mc-15-2-st-508-1836079)      |

> | BAT (20-25 V) |     4 A      |   100 W   | [Phoenix Contact MSTB 2,5/ 2-ST-5,08](https://www.phoenixcontact.com/pl-pl/produkty/wtyk-plytki-drukowanej-mstbt-25-2-st-508-1779987) |

Reviewer: ```suggestion
| BAT (18-25.2 V) |     4 A      |   100 W   | [Phoenix Contact MSTB 2,5/ 2-ST-5,08](https://www.phoenixcontact.com/pl-pl/produkty/wtyk-plytki-drukowanej-mstbt-25-2-st-508-1779987) |
```

    light: useBaseUrl('/img/robots/raph/specification/Top_plate_black.webp'),
    dark: useBaseUrl('/img/robots/raph/specification/Top_plate_white.webp'),
  }}
  width="3000"
  height="2121"
  figStyle={{
    width: 800,
  }}
/>

<CenterContent>

| Parameter                             |                                              Value                                               |
| ------------------------------------- | :----------------------------------------------------------------------------------------------: |
| Payload capacity                      |                                              10 kg                                               |
| Hole grid spacing                     |                                          25 mm x 25 mm                                           |
| Mounting hole dimensions              | 5.4 mm, compatible with: <br /> M5 screws <br/> [CLS-M4-1 press-in nuts](https://mou.sr/3VyUg39) |
| Mounting plate dimensions (L x W x D) |                                      330 mm x 198 mm x 2 mm                                      |

</CenterContent>

:::info

Apart from the mounting plate, it is possible to mount payloads directly to

> v-slot extrusions of the Raph Rover. Effectively providing a **360 mm by 430

Reviewer: ```suggestion
v-slot extrusions of the Raph Rover, effectively providing a **360 mm by 430
```

mm** working area.

:::

## Software & Control

<ThemedImageZoom
  alt="Overview of Raph Rover's software components"
  sources={{
    light: useBaseUrl(
      '/img/robots/raph/specification/raph-software-light.webp',
    ),
    dark: useBaseUrl('/img/robots/raph/specification/raph-software-dark.webp'),
  }}
  width="2000"
  height="819"
  figStyle={{
    paddingBottom: '4rem',
  }}
/>

<CenterContent>

| Parameter        |                             Value                              |
| ---------------- | :------------------------------------------------------------: |

> | Operating System |                        Ubuntu 22.04 LTS                        |

Reviewer: ```suggestion
| Operating System |                        Ubuntu 24.04 LTS                        |
```

| Core Framework   |                          ROS 2 Jazzy                           |
| User Interface   |              Web-based GUI accessible over Wi-Fi               |

> | UI Features      | Real-time teleoperation, H.264 video stream, system monitoring |

Reviewer: We don't encode it to H.264 (at least yet). It's mjpeg now.
```suggestion
| UI Features      | Real-time teleoperation, live video streaming, system monitoring |
```
